SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Utils/CultureInfoHelper.cs

```diff
@@ -15,7 +16,7 @@ class CultureInfoHelper
         private const string LOCALE_NAME_SYSTEM_DEFAULT = "!x-sys-default-locale";
 
         private const int BUFFER_SIZE = 530;
-        
+
         [DllImport("api-ms-win-core-localization-l1-2-0.dll", CharSet = CharSet.Unicode)]
         private static extern int GetLocaleInfoEx(string lpLocaleName, uint LCType, StringBuilder lpLCData, int cchData);
```

Comment:
Lines 31/32: please update code and comment to

```
                    // If system/user default doesn't work, use current culture which is app locale.
                    return CultureInfo.CurrentCulture;
```

To avoid sending empty string as fallback.